scripts/reloader.bash

```diff
@@ -20,9 +30,10 @@ fi
 
 
 if [ ! -z "${2}" ] && [[ "${2}" =~ ^(aliases|completion|plugins)$ ]] && [ -d "${2}/enabled" ]; then
-  # TODO: We should warn users they're using legacy enabling
+  _log_warning "Using legacy enabling, please update your bash-it version and migrate"
```

Comment:
Please include the `$2` parameter in the log message, it would be interesting to know for which type of components this is triggered.

Reply:
done